grammar.js

```diff
@@ -217,7 +217,7 @@ module.exports = grammar({
       $.braced_variable
     ),
 
-    braced_variable: $=> /\${[^}]+}/,
+    braced_variable: $=> /\$\{[^}]+\}/,
```

Comment:
I had to manually escape these according to [this change](https://github.com/tree-sitter/tree-sitter/pull/2838)